packages/app/src/vis-packs/nexus/NxSignalPicker.tsx

```diff
@@ -0,0 +1,35 @@
+import type { ComplexType, NumericType } from '@h5web/shared/hdf5-models';
+
+import type { DatasetDef } from './models';
+import styles from './NxSignalPicker.module.css';
+
+interface Props<T extends NumericType | ComplexType> {
+  definitions: DatasetDef<T>[];
+  onChange: (def: DatasetDef<T>) => void;
+}
+
+function NxSignalPicker<T extends NumericType | ComplexType>(props: Props<T>) {
+  const { definitions, onChange } = props;
+
+  return (
+    <p className={styles.bar}>
+      <span className={styles.text}>Signal:</span>
+      {definitions.map((def, index) => (
+        <label key={def.dataset.name} className={styles.label}>
+          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Pardon my French, but this rule is so dumb... I might disable it one day...

Reply:
Yeah, it also got in my way a few times